kit/scrolls/android/src/main/java/tonlabs/uikit/scrolls/UIKitAccordionOverlayViewManager.java

```diff
@@ -0,0 +1,131 @@
+
+    @NonNull
+    @Override
+    public String getName() {
+        return UIKitAccordionOverlayView.REACT_CLASS;
+    }
+
+    @NonNull
+    @Override
+    protected UIKitAccordionOverlayView createViewInstance(@NonNull ThemedReactContext reactContext) {
+        return new UIKitAccordionOverlayView(reactContext);
+    }
+
+    // MARK:- commands
+
+    @Override
+    public Map<String, Integer> getCommandsMap() {
+        return MapBuilder.of(
+                COMMAND_SHOW_KEY,
+                COMMAND_SHOW_ID,
+                COMMAND_MOVE_AND_HIDE_KEY,
+                COMMAND_MOVE_AND_HIDE_ID
+        );
+    }
+
+    @Override
+    public void receiveCommand(@NonNull @NotNull UIKitAccordionOverlayView root,
+                               int commandId,
+                               @Nullable ReadableArray args) {
+        Assertions.assertNotNull(root);
+        Assertions.assertNotNull(args);
+        switch (commandId) {
+            case COMMAND_SHOW_ID:
+            {
+                root.show(args);
+                root.dispatchEvent(COMMAND_SHOW_KEY);
+                return;
+            }
+            case COMMAND_APPEND_ID:
+            {
+                root.append(args);
+                root.dispatchEvent(COMMAND_APPEND_KEY);
+                return;
+            }
+            case COMMAND_MOVE_AND_HIDE_ID:
+            {
+                root.moveAndHide(args);
+                return;
+            }
+
+            default:
+                throw new IllegalArgumentException(
+                        String.format(
+                                "Unsupported command %d received by %s.",
+                                commandId, this.getClass().getSimpleName()));
+        }
+    }
+
+    @Override
+    public void receiveCommand(@NonNull @NotNull UIKitAccordionOverlayView root,
+                               String commandKey,
+                               @Nullable ReadableArray args) {
+        Assertions.assertNotNull(root);
+        Assertions.assertNotNull(args);
+        switch (commandKey) {
+            case COMMAND_SHOW_KEY:
+            {
+                root.show(args);
+                root.dispatchEvent(COMMAND_SHOW_KEY);
+                return;
+            }
+            case COMMAND_APPEND_KEY:
+            {
+                root.append(args);
+                root.dispatchEvent(COMMAND_APPEND_KEY);
+                return;
+            }
+            case COMMAND_MOVE_AND_HIDE_KEY:
+            {
+                root.moveAndHide(args);
+                return;
+            }
+
+            default:
+                throw new IllegalArgumentException(
+                        String.format(
+                                "Unsupported command %s received by %s.",
+                                commandKey, this.getClass().getSimpleName()));
+        }
+    }
+
+    // MARK:- Events
+
+    @Override
+    public @Nullable Map<String, Object> getExportedCustomDirectEventTypeConstants() {
```

Comment:
Do not understand what is this for... :)

Reply:
https://reactnative.dev/docs/native-components-android#events

Reply:
Cool, I've learned smth new today ;)

Reply:
Glad I motivated you to do that :D